doc/src/build/fullnode.md

```diff
@@ -288,3 +288,85 @@ If you followed the instructions for [Building from Source](#building-from-sourc
     ```
 
 Your Full node starts on: `http://127.0.0.1:9000`.
+
+# Pruning 
+
+## Object Pruning
+
+Object pruning involves the removal of older object version from the db. New object versions are added to the db during transaction execution (making older versions ready for garbage collection) and can become a problem if not pruned, as it can slow down the performance of the db and take up too much disk space. Object pruning works in the background by tailing checkpoint stream where each checkpoint contains object versions which are now eligible for pruning.
```

Comment:
easier to diff paragraphs with line breaks instead of spaces. They should just show up as spaces in the final formatting
```suggestion
Object pruning involves the removal of older object versions from the fullnode db.
New object versions that are added to the db during transaction execution (making older versions ready for garbage collection) can become a problem if not pruned. Many old object versions can slow down the performance of the db and take up large amounts of disk space.
Object pruning works in the background by tailing the checkpoint stream, identifying each checkpoint which contains object versions that are eligible for pruning.
```